tests/suites/firewall/task.sh

```diff
@@ -8,16 +8,21 @@ test_firewall() {
 
 	setup_awscli_credential
 
+	setup_gcloudcli_credential
+
 	echo "==> Checking for dependencies"
-	check_dependencies juju aws
+	check_dependencies juju aws gcloud
 
 	file="${TEST_DIR}/test-firewall.txt"
 
 	bootstrap "test-firewall" "${file}"
 
 	case "${BOOTSTRAP_PROVIDER:-}" in
 	"ec2")
-		test_firewall_ssh
+		test_firewall_ssh_ec2
+		;;
+	"gce")
```

Comment:
Update the comment below. This test does run on GCP, not just AWS.

`		echo "==> TEST SKIPPED: test_firewall_ssh test runs on aws only"`